src/frameserver/PointGreyCam.cpp

```diff
@@ -282,7 +282,7 @@ void PointGreyCam<T>::connectToCamera(int index)
     pg::Error error = busMgr.GetCameraFromIndex(index, &guid);
     if (error != pg::PGRERROR_OK)
         throw (rte(error.GetDescription()));
-
+    pg::Camera camera_;
```

Comment:
camera_ is declared in the header at line 84, no need to re-declare here. Additionally, this class is now templated around camera of type `T` which can be either a `pg::GigeGamera` or a `pg::Camera`.